setup/azurecmdfiles.wxi

```diff
@@ -204,6 +204,9 @@
                         <Component Id="cmp048B605B1072F4A5467F2AFFD8ED2459" Guid="*">
```

Comment:
Please revert these changes.  None of the dll-Help.psd1 files should be part of the installer package.